pgr-services/src/main/java/org/egov/pgr/service/MigrationService.java

```diff
@@ -216,14 +254,14 @@ private void setStatusToUUIDMap(){
          * Transform address and set geo location
          */
         GeoLocation geoLocation = GeoLocation.builder().longitude(longitutude).latitude(latitude).build();
-        org.egov.pgr.web.models.Address address = transformAddress(serviceV1.getAddressDetail());
-        address.setGeoLocation(geoLocation);
-        address.setTenantId(tenantId);
+        log.info("Address details: " + serviceV1.getAddressDetail());
+        org.egov.pgr.web.models.Address address = null;
+        if(serviceV1.getAddressDetail() != null) {
```

Comment:
how many addressDetails are null?

Reply:
none of them are, actually earlier while testing some of the locality values were null, so I have handled that in the code itself so this "if" check is redundant, will remove it.